### ISSUE TITLE
not valid package symfony/.github 

### ISSUE THREAD

**Issue by Punk-UnDeaD** [symfony/flex#802]
symfony/.github not found on packagist
https://packagist.org/packages/symfony/.github

**Comment by chapterjason** [comment]
It could be because this package does not exist.

**Comment by wouterj** [comment]
That repository is not meant to be used (and only provides default GitHub templates).

Did you get this error when working on a Symfony application?

**Comment by Punk-UnDeaD** [comment]
@wouterj `composer req .github` throw [InvalidArgumentException] Could not find package symfony/.github.

**Comment by stof** [comment]
@fabpot could this be a bug of the Flex server generating a `.github` alias due to the existence of the repo even if this is not an actual package ?

@Punk-UnDeaD next question: why are you running this command ? What would you expect ?

**Comment by wouterj** [comment]
Yes, this is indeed a bug: https://flex.symfony.com/aliases.json

However, I don't think the error is wrong: The package indeed doesn't exists and that's not a bug but a fact :)
